analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/SymbolicContextTest.cs

```diff
@@ -31,27 +31,27 @@ public class SymbolicContextTest
     [TestMethod]
     public void NullArgument_State_Throws()
     {
-        var create = () => new SymbolicContext(CreateOperation(), null, false, 0, Array.Empty<ISymbol>());
+        var create = () => new SymbolicContext(null, CreateOperation(), null, false, 0, Array.Empty<ISymbol>());
         create.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("state");
     }
 
     [TestMethod]
     public void NullArgument_CapturedVariables_Throws()
     {
-        var create = () => new SymbolicContext(CreateOperation(), ProgramState.Empty, false, 0, null);
+        var create = () => new SymbolicContext(null, CreateOperation(), ProgramState.Empty, false, 0, null);
         create.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("capturedVariables");
     }
 
     [TestMethod]
     public void NullOperation_SetsOperationToNull() =>
-        new SymbolicContext(null, ProgramState.Empty, false, 0, Array.Empty<ISymbol>()).Operation.Should().Be(null);
+        new SymbolicContext(null, null, ProgramState.Empty, false, 0, Array.Empty<ISymbol>()).Operation.Should().Be(null);
 
     [TestMethod]
     public void PropertiesArePersisted()
     {
         var operation = CreateOperation();
         var state = ProgramState.Empty.SetOperationValue(operation, SymbolicValue.Empty);
-        var sut = new SymbolicContext(operation, state, true, 42, Array.Empty<ISymbol>());
+        var sut = new SymbolicContext(null, operation, state, true, 42, Array.Empty<ISymbol>());
```

Comment:
This one needs more elaborate scaffolding and assertion, see the test name `PropertiesArePersisted`